source/help/apps/desktop-guide.rst

```diff
@@ -4,7 +4,7 @@ Using Mattermost Desktop App
 
 Mattermost desktop applications are available for Windows, Mac and Linux operating systems. 
 
-You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up.
+You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up and for troubleshooting tips.
 
```

Comment:
Is the url https://docs.mattermost.com/install/desktop.html correct? I don't see any troubleshooting info on that page.

Is the URL https://docs.mattermost.com/help/apps/desktop-guide.html?